.circleci/config.yml

```diff
@@ -46,6 +46,7 @@ workflows:
                 - "1.15"
                 - "1.16"
                 - "1.17"
+                - "1.18"
```

Comment:
Gosh, I think I like our [one-liner matrix-params pattern for versions](https://github.com/honeycombio/libhoney-go/pull/172/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R23) better.